diagnostics/surface_albedo_feedback/surface_albedo_feedback_calcs.py

```diff
@@ -0,0 +1,242 @@
+    albedo_abmod_file=output_dir+'surface_albedo_abrupt-4xCO2.nc'
+
+
+    # process model piControl data to compute kerel, albedo, and Tglob
+    TAS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{TAS_var}.piControl.nc".format(**os.environ)
+    area_input_file = "{DATADIR}/mon/areacella_fx_{model}_abrupt-4xCO2_r1i1p1f1_gn.nc".format(**os.environ)
+    tas=xr.open_dataset(TAS_input_file)
+    fx=xr.open_dataset(area_input_file)
+    tas=tas[TAS_var]
+    fx=fx['areacella']
+    Tglob=globaltimemean(tas, fx)
+
+    # process model piControl calc kernel and albedo
+    piFSDT_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSDT_var}.piControl.nc".format(**os.environ)
+    piFSDS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSDS_var}.piControl.nc".format(**os.environ)
+    piFSUT_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSUT_var}.piControl.nc".format(**os.environ)
+    piFSUS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSUS_var}.piControl.nc".format(**os.environ)
+
+    dt = readandclimo(FSDT_var,piFSDT_input_file)
+    ds = readandclimo(FSDS_var,piFSDS_input_file)
+    ut = readandclimo(FSUT_var,piFSUT_input_file)
+    us = readandclimo(FSUS_var,piFSUS_input_file)   
+
+    kernel,albedo=kernelalbedo(dt, ds, ut, us)
+    pimod=xr.merge([kernel,albedo,Tglob])
+    pimod.to_netcdf(kernel_pimod_file)
+
+    # process model abrupt-4xCO2 data to compute albedo, and Tglob
+    TAS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{TAS_var}.abrupt-4xCO2.nc".format(**os.environ)
+    area_input_file = "{DATADIR}/mon/areacella_fx_{model}_abrupt-4xCO2_r1i1p1f1_gn.nc".format(**os.environ)
+    tas=xr.open_dataset(TAS_input_file)
+    fx=xr.open_dataset(area_input_file)
+    tas=tas[TAS_var]
+    fx=fx['areacella']
+    Tglob=globaltimemean(tas, fx)
+
+    abFSDS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSDS_var}.abrupt-4xCO2.nc".format(**os.environ)
+    abFSUS_input_file = "{DATADIR}/mon/{CASENAME}.mon.{FSUS_var}.abrupt-4xCO2.nc".format(**os.environ)
+    ds = readandclimo(FSDS_var,abFSDS_input_file)
+    us = readandclimo(FSUS_var,abFSUS_input_file)   
+    albedo = us/ds
+    albedo.name = 'albedo'
+    abmod=xr.merge([albedo,Tglob])
+    abmod.to_netcdf(albedo_abmod_file)
+
+    # historical kernel and albedo from model and CERES
+    FSDT_input_file = "{DATADIR}/mon/{CASENAME}.{FSDT_var}.mon.nc".format(**os.environ)
+    FSDS_input_file = "{DATADIR}/mon/{CASENAME}.{FSDS_var}.mon.nc".format(**os.environ)
+    FSUT_input_file = "{DATADIR}/mon/{CASENAME}.{FSUT_var}.mon.nc".format(**os.environ)
+    FSUS_input_file = "{DATADIR}/mon/{CASENAME}.{FSUS_var}.mon.nc".format(**os.environ)
+
+    dt = readandclimo(FSDT_var,FSDT_input_file)
+    ds = readandclimo(FSDS_var,FSDS_input_file)
+    ut = readandclimo(FSUT_var,FSUT_input_file)
+    us = readandclimo(FSUS_var,FSUS_input_file)   
+
+    kernel,albedo=kernelalbedo(dt, ds, ut, us)
+    histmod=xr.merge([kernel,albedo])
+    histmod.to_netcdf(kernel_histmod_file)
+
+    processraw=False
```

Comment:
Since it is not used, it makes more sense to me to replace the entire code block with the comment that the albedo calculation is located in the kernal_obs_file.